### DISCUSSION TITLE
执行过程很完美，没有错误，但我找不到文件

### DISCUSSION THREAD

**Discussion by guicaiyue** [upptime/upptime#502] [Q&A]
我配置 Deploy keys and Secrets 在一个私有仓库
执行过程没有问题，但我并未在私有仓库内看到新的分支上传的代码。
我最开始尝试把 Deploy keys 配置到另一个共有仓库，但也位找到它，我只能在job日志中看到构建产物的存在

**Comment by aschenmaker** [comment]
没有太懂你的意思，希望可以描述的详细一些
> https://guicaiyue.github.io/uptime/
